03-transactions.md

```diff
@@ -686,21 +780,38 @@ The *expected weight* of a commitment transaction is calculated as follows:
 			....htlc_output's...
 		- lock_time: 4 bytes
 
+	 commitment_transaction (option_anchor_outputs): 223 + 43 * num-htlc-outputs bytes
+		- version: 4 bytes
+		- witness_header <---- part of the witness data
+		- count_tx_in: 1 byte
+		- tx_in: 41 bytes
+			funding_input
+		- count_tx_out: 1 byte
+		- tx_out: 172 + 43 * num-htlc-outputs bytes
+			output_paying_to_remote,
+			output_paying_to_local,
+			output_anchor,
+			output_anchor,
+			....htlc_output's...
+		- lock_time: 4 bytes
+
 Multiplying non-witness data by 4 results in a weight of:
 
-	// 500 + 172 * num-htlc-outputs weight
+	// 500 + 172 * num-htlc-outputs weight (no option_anchor_outputs)
+	// 892 + 172 * num-htlc-outputs weight (option_anchor_outputs)
```

Comment:
How do you get 892 here ? 

43 * 4 * 2 + 500 = 844 ?

Reply:
The `to_remote` output has changed from p2wkh to p2wsh (+12 bytes) because it now has CSV 1 on it.